File: files/400-499/422/CH-422.R
library(tidyverse)
library(readxl)
library(matricks)

path <- "400-499/422/CH-422 Matrix Normalization.xlsx"
input <- read_excel(path, range = "C4:F7", col_names = FALSE) %>%
  as.matrix()
test <- read_excel(path, range = "J4:M7", col_names = FALSE) %>%
  as.matrix()

result = input - antidiag(input)
all.equal(result, test)
#> [1] TRUE

File: files/400-499/422/CH-422.py
import pandas as pd

path = "400-499/422/CH-422 Matrix Normalization.xlsx"
input = pd.read_excel(path, usecols="C:F", nrows=4, skiprows=3, header=None)
test = pd.read_excel(path, usecols="J:M", nrows=4, skiprows=3, header=None)

result = input.subtract(input.to_numpy()[:, ::-1].diagonal(), axis=0)
result.columns = test.columns

print(result.equals(test))
#> True